include/swift/Frontend/FrontendOptions.h

```diff
@@ -369,6 +375,9 @@ class FrontendOptions {
   /// Emit remarks indicating use of the serialized module dependency scanning cache.
   bool EmitDependencyScannerCacheRemarks = false;
 
+  /// The patch at which the dependency scanner can write generated files.
+  std::string ScannerOutputDir;
```

Comment:
Hmm, I don't think the scanner should be writing out any generated files. Rather, it should communicate any files that need generating to the client, i.e. SwiftDriver, and then have the client do the required IO.

Reply:
Given that for CAS purposes we do need to have the scanner have the information of where the chained bridging header will ultimately be serialized to, we could add an option for this specifically: `-chained-bridging-header-output-path` that the build system client can provide.